std/algorithm/searching.d

```diff
@@ -5005,6 +5005,31 @@ Otherwise if set to `OpenRight.no`, then the interval is closed to the right
 (last element included).
  */
 alias OpenRight = Flag!"openRight";
+/**
+New Interval option specifier for `until` (below) and others.
+
+If set to `sentinelExcluded`, then only the part of the range before the
+match is included.
+
+If set to `sentinelIncluded`, then the part of the range up
+to the end of the match is included.
+
+If set to `firstPartOfSentinelIncluded`, then the part of the range up
+to the and including the start of the match is included.
```

Comment:
> then the part of the range up to the and including the start of the match is included.

Should be:

then the part of the range up to, and including, the start of the match is included.